CHANGELOG.md

```diff
@@ -14,6 +14,27 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `XRPPayIDClientInterface` was deprecated for two releases and has been removed. Use `XrpPayIdClient` instead.
 - `XRPPayIDClient` was deprecated for two releases and has been removed.  Use `XrpPayIdClient` instead.
 - `XRPLNetwork` was deprecated for two releases and has been removed.  Use `XrplNetwork` instead.
+- `memos` on `SendXrpDetails` was deprecated for two releases and has been removed. Use `memoList` instead.
```

Comment:
Could we write an overarching note in the changelog? That way it's more obvious what is happening, rather than a laundry list of changes? 

This release of Xpring SDK formalizes the casing of acronomyms. Previously, if you used classes called `XRPFoo` or `PayIDBar`, they are now called `XrpFoo` and `PayIdBar`.